commit 61619523988c46c7882bea270c810ef794743324 (refs/changes/24/724/2)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2016-10-18T13:39:09-04:00 (3 years ago)
    
    RELENG-612 prefer (git+)https urls
    Reviewed by: Trent Mick <trent.mick@joyent.com>
    Approved by: Trent Mick <trent.mick@joyent.com>

diff --git a/.gitmodules b/.gitmodules
index 556c791..1c7169f 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -1,12 +1,12 @@
 [submodule "deps/restdown"]
 	path = deps/restdown
-	url = git://github.com/trentm/restdown.git
+	url = https://github.com/trentm/restdown.git
 [submodule "deps/javascriptlint"]
 	path = deps/javascriptlint
-	url = git://github.com/davepacheco/javascriptlint.git
+	url = https://github.com/davepacheco/javascriptlint.git
 [submodule "deps/jsstyle"]
 	path = deps/jsstyle
-	url = git://github.com/davepacheco/jsstyle.git
+	url = https://github.com/davepacheco/jsstyle.git
 [submodule "deps/zfs_snapshot_tar"]
 	path = deps/zfs_snapshot_tar
-	url = git://github.com/joyent/zfs_snapshot_tar.git
+	url = https://github.com/joyent/zfs_snapshot_tar.git
diff --git a/package.json b/package.json
index 5f180b8..0b817b7 100644
--- a/package.json
+++ b/package.json
@@ -16,11 +16,11 @@
     "kstat": "0.5.0",
     "lazy-property": "1.0.0",
     "libuuid": "0.1.4",
-    "lockfd": "git+ssh://git@github.com:joyent/node-lockfd.git#eeec5216dd493b44a870d0369872f56ab0788d0f",
+    "lockfd": "git+https://github.com/joyent/node-lockfd.git#eeec5216dd493b44a870d0369872f56ab0788d0f",
     "mkdirp": "^0.3.4",
     "nodeunit": "0.9.1",
     "once": "1.3.0",
-    "pty.js": "git://github.com/joshwilsdon/pty.js.git#exitStatus",
+    "pty.js": "git+https://github.com/joshwilsdon/pty.js.git#exitStatus",
     "restify": "4.0.0",
     "rimraf": "2.4.3",
     "sdc-clients": "~9.1",
@@ -28,12 +28,12 @@
     "semver": "2.2.1",
     "shell-escape": "^0.2.0",
     "sprintf": "0.1.1",
-    "trace-event": "git+ssh://git@github.com:joyent/node-trace-event.git#9f7d00b8c3594def9ef534f68c16c215c3dba9f2",
-    "tracker": "git+ssh://git@github.com:joyent/node-tracker.git#3a139906c9eb8d8684ac54cf54cd010315856042",
+    "trace-event": "git+https://github.com/joyent/node-trace-event.git#9f7d00b8c3594def9ef534f68c16c215c3dba9f2",
+    "tracker": "git+https://github.com/joyent/node-tracker.git#3a139906c9eb8d8684ac54cf54cd010315856042",
     "vasync": "^1.6.3",
     "verror": "1.6.0",
-    "vmadm": "git+ssh://git@github.com:joyent/node-vmadm.git#94e95c6b96c2c5947281a5ea3df763928a65219c",
-    "zfs": "git+ssh://git@github.com:joyent/node-zfs.git#657a90d9424c45066e3e0919dfe9b34f5636e0e9"
+    "vmadm": "git+https://github.com/joyent/node-vmadm.git#94e95c6b96c2c5947281a5ea3df763928a65219c",
+    "zfs": "git+https://github.com/joyent/node-zfs.git#657a90d9424c45066e3e0919dfe9b34f5636e0e9"
   },
   "sdcDependencies": {
     "config-agent": ">=1.3.0"
